src/kernel/src/fs/dev/console.rs

```diff
@@ -55,3 +67,17 @@ impl Display for Console {
         Self::PATH.fmt(f)
     }
 }
+
+#[derive(Debug, Error)]
+enum IoctlErr {
```

Comment:
I never saw any error types in std having `Err` as a suffix.

Reply:
I'ff fix it.

Reply:
Fixed